.github/ISSUE_TEMPLATE/proposal-without-explainer.md

```diff
@@ -0,0 +1,43 @@
+---
+name: New Proposal Without an Explainer
+about: If you've got a new idea, but have not created an Explainer elsewhere, use this template
+title: 'Proposal: PROPOSAL NAME'
+labels: no-explainer
+assignees: ''
+---
+
+## Introduction
+
+*Describe the challenge or problem on the web you are proposing we address.*
+
+## Use Cases (Recommended)
+
+*Clearly define one (or, ideally, more) ways in which this challenge or problem is blocking real-world applications. Please be as specific as you can.* 
+
+## Goals (Optional)
+
+*A list of what this proposal must do in order to be considered successful.*
+
+## Non-goals (Optional)
+
+*A list of aspects of this challenge/problem or related challenges/problems that this proposal does not need to address.*
+
+## Proposed Solution
+
+*Without being too prescriptive, articulate a specific way (procedurally, algorithmically, declaratively) that a new or current web technology solves an existing problem or challenge.*
+
+## Examples (Recommended)
+
+*Share one or more examples of how your proposal would be used by developers.*
+
+## Alternate Approaches (Optional)
+
+*Could this challenge be addressed another way? Including other options you considered can help others understand the limitations or adverse side effects of other approaches.*
+
+## Privacy & Security Considerations
+
+*Perform a [self-review](https://w3ctag.github.io/security-questionnaire/). You may think of additional privacy/security considerations other than the ideas listed. If you find no considerations after careful review: "No considerable privacy or security concerns are expected, but we welcome community feedback."*
+
+## Open Questions (Optional)
```

Comment:
I like this section a lot--an encouragement to provide specific feedback. I would title it "Let's Discuss..." :) (but "Open Questions" works too).

Reply:
Done.